rust/repositories.bzl

```
@@ -59,6 +59,16 @@ def rules_rust_dependencies():
         url = "https://github.com/bazelbuild/apple_support/releases/download/0.11.0/apple_support.0.11.0.tar.gz",
     )
 
+    # process_wrapper needs a low-dependency way to process json.
+    maybe(
+        http_archive,
+        name = "tinyjson",
```

How do folks feel about scoping the name to avoid it impacting users?
```suggestion
        name = "rules_rust_tinyjson",
```